playbooks/rook/rook.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Apply role rook
+  hosts: localhost
+  connection: local
+
+  roles:
+    - role: osism.services.rook
```

Comment:
I see that role (https://github.com/osism/ansible-collection-services/pull/1427) does not define any `rook_storage_nodes` or similar settings, hence if I test this play on Testbed I do not get any OSDs.

I'm curious if the current configuration is the intended final state or if there are plans to further customize the default options for the osism.services.rook role, particularly for Testbed deployment.

Reply:
Have a look at https://github.com/osism/testbed/blob/rook/environments/rook/configuration.yml. This will add the missing rook environment + the mentioned configuration parameter.

Reply:
thx

Reply:
@matofeder @berendt The idea is that even in the default configuration, nothing potentially dangerous (e.g. deleting disks) is configured. This is why you NEED the config repo (e.g. testbed repo) to run this.

Reply:
Makes sense from my point of view. We have made this explicit in the configuration repository for ceph-ansible for exactly this reason.